src/audio-worklet/free-queue/src/interface/free_queue.h

```diff
@@ -1,24 +1,25 @@
-#ifndef FREE_QUEUE_C
-#define FREE_QUEUE_C
+#ifndef FREE_QUEUE_C_H_
+#define FREE_QUEUE_C_H_
 
-
-#include <stdint.h>
-#include <stdlib.h>
 #include <stdatomic.h>
+#include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
-#include <stdbool.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#ifndef EMSCRIPTEN_KEEPALIVE
-#define EMSCRIPTEN_KEEPALIVE __attribute__((used))
-#endif
-
 /**
- * C Struct to represent FreeQueue
+ * C Struct representation of FreeQueue.
+ * It has 4 data members.
+ * bufferLength stores size/length of backing buffer.
```

Comment:
Or you can even follow JSDoc notation. It'll be much clearer. The Chromium style is to add a comment for each member right above the code.

Reply:
This comment doesn't seem to be addressed yet.